operate/webapp/src/main/java/io/camunda/operate/webapp/security/auth/AuthUserService.java

```diff
@@ -23,21 +24,21 @@
       + OperateProfileService.SSO_AUTH_PROFILE
       + " & !"
       + OperateProfileService.IDENTITY_AUTH_PROFILE
-      + " & !"
```

Comment:
@maryarm can you help me to understand this part of removing the profile for Operate? 

We have something similar here on Tasklist: https://github.com/camunda/camunda/blob/fc225e2f15a9d08ad519f4edd26b0b709b0a1154/tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/security/se/SearchEngineUserDetailsService.java#L35

Reply:
@marcosgvieira, I saw in the history that The AUTH_BASIC profile has added to AuthUserService by Ben. The reason is: after we have enabled auth basic in camunda-authentication, principal type is CamundaUser. As now I changed the code to always return CamundaUser as principal no matter which UserDetailService is used, I removed the profile exclusion here.

Reply:
so, we need to remove `!AUTH_BASIC` from `SearchEngineUserDetailsService` in Tasklist module as well?

Reply:
no, don't think so, I removed the AUTH_BASIC from AuthUserService in operate not form UserDetailsServices, so it could keep the profile, I have just changed loadByUsername to return CamundaUser.

Reply:
I see, it should be kept for the Components extending `UserDetailsService`